PanoptesNetClient/PanoptesNetClient/ApiClient.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace PanoptesNetClient
+{
+    public class ApiClient
+    {
+        static HttpClient client = new HttpClient();
+        private static ApiClient instance;
+
+        private ApiClient()
+        {
+            RunAsync().GetAwaiter().GetResult();
+        }
+
+        public static ApiClient Instance
+        {
+            get
+            {
+                if (instance == null)
+                {
+                    instance = new ApiClient();
+                }
+                return instance;
+            }
+        }
+
+        static async Task RunAsync()
```

Comment:
This naming here is kind of confusing. From what I can tell this code should just be initializing the http client because it is called when the client is created.

Maybe a name like `InitializeClient` would be better

Reply:
Also, I think this should be `private` instead of `static`

Reply:
Good point. I plan on changing up a good bit of this once I flesh the client out, but it's good practice to start doing it now.